site/src/pages/CreateWorkspacePage/CreateWorkspacePageViewExperimental.tsx

```diff
@@ -195,10 +205,76 @@ export const CreateWorkspacePageViewExperimental: FC<
 		presetOptions,
 		selectedPresetIndex,
 		presets,
-		parameters,
 		form.setFieldValue,
+		parameters,
 	]);
 
+	const [debouncedTimer, setDebouncedTimer] = useState<NodeJS.Timeout | null>(
```

Comment:
super pedantic but `NodeJS.Timeout` is not the right type here, because for browsers `setTimeout` returns `number`. the way I like to work around messy types for this is `ReturnType<typeof setTimeout>`.